R/get.R

```diff
@@ -95,6 +102,40 @@ docdb_get.src_mongo <- function(src, key, limit = NULL, ...) {
   jsonlite::stream_in(file(dump), verbose = FALSE)[,-1]
 }
 
+#' @export
+docdb_get.src_sqlite <- function(src, key, limit = NULL, ...) {
+
+  assert(key, "character")
+  assert(limit, "integer")
```

Comment:
probably good idea to require the `limit` param to be an integer. however, we'd need it to be consistent across the methods in this fxn. in your example for this method, you pass in a numeric value, so the example fails. the other examples that use a limit param in this manual file will need to be changed as well